src/entity-manager/EntityManager.ts

```diff
@@ -580,7 +580,7 @@ export class EntityManager {
         } else if (maybeOptions && FindOptionsUtils.isFindOneOptions(maybeOptions) && maybeOptions.join) {
             alias = maybeOptions.join.alias;
         }
-        const qb = this.createQueryBuilder(entityClass, alias);
+        const qb = this.createQueryBuilder(entityClass, alias, this.queryRunner);
```

Comment:
I think you don't need to pass `this.queryRunner` here since `this.createQueryBuilder` already uses it.